### DISCUSSION TITLE
EMU E5000 ULTRA - working

### DISCUSSION THREAD

**Discussion by baritonomarchetto** [keirf/flashfloppy#577] [General]
This is just to report that I succesfully installed a Gotek emulator with flash floppy in my E5000. It's very likely that an E6400 ULTRA, and E4XT will work too.

Jumpers must be placed at S0 and JC, like for ESI-32.

If jumpers are not installed this way, the OS will freeze during the first bootup sequence.

Please Keirf, add this info to the "Host Platforms" list (the existing "EMU ESI-32" platform should be changed to "EMU" and the two emulators splitted or something like that ;))

**Comment by tizzulo** [comment]
Not the same config I settled on, but if you can load, save and format, then I think your method is easiest.

**Comment by L50six** [comment]
After a evening franticly searching the net, and finding this post I can confirm that these jumper settings also work on an EMU-E6400 with EOS 3.00b (Gotek with FF 3.38)

Default settings (S1) would not hang during boot but make the whole sampler incredibly sluggish and unusable. Thank you @baritonomarchetto!

@tizzulo But still I am curious as to what your config is then? ;)
